source/fundamentals/crud/write-operations/modify.txt

```diff
@@ -13,11 +13,23 @@ Modify Documents
 Overview
 --------
 
-You can modify documents in a MongoDB collection using either **update**
-or **replace** operations. Update operations mutate
+You can modify documents in a MongoDB collection using **update**
+and **replace** operations. Update operations change
```

Comment:
Issue:
I think this sentence might be inaccurate as it mentions updating fields but omits values, which is probably the more common use case.

Suggestion:
"Update operations modify the fields and values of a document while keeping the other fields and values."